src/openshift/component.ts

```diff
@@ -90,6 +91,11 @@ export class Component extends OpenShiftItem {
         return !!ws;
     }
 
+    @vsCommand('openshift.component.open.devfileRegistry')
```

Comment:
It should be moved to componentTypesView.ts and called from context menu or inline group on Registry tree item.

Reply:
Even better to keep it in registryViewLoader.ts.